index.html

```diff
@@ -57,10 +60,313 @@ <h1>I'm Salma <br> Glad to see you!</h1>
+   
+    </section> 
+</section>
+   
+<!--Work Cards Section-->
+
+<!--About Me Section-->
+
+<section class="About-section">
+   <div class="about-wrap">
+    <h1>About <br> MySelf</h1>
+    <p>Hello I'm a software developer!
+         I can help you build a product , 
+         feature or website Look through some 
+         of my work and experience! If you like 
+         what you see and have a project you need coded,
+         don't hestiate to contact me.
+    </p>
+    <div class="link-text">
+        <a href="#">LETT'S CONNECT</a>
+    </div>
+    
+    <ul class="social-media">
+        <li>
+         <a href="" target="_blank" rel="noopener noreferrer">
+             <img class="s-m-icon" src="images/Vector.svg" alt="social-icon">
+         </a>
+        </li>
+
+        <li>
+         <a href="" target="_blank" rel="noopener noreferrer">
+             <img class="s-m-icon" src="images/Linkedin icon.svg" alt="social-icon">
+         </a>
+        </li>
+
+        <li>
+         <a href="" target="_blank" rel="noopener noreferrer">
+             <img class="s-m-icon" src="images/Vector (1).svg" alt="social-icon">
+         </a>
+        </li>
+
+        <li>
+         <a href="" target="_blank" rel="noopener noreferrer">
+             <img class="s-m-icon" src="images/Vector (2).svg" alt="social-icon">
+         </a>
+        </li>
+
+        <li>
+         <a href="" target="_blank" rel="noopener noreferrer">
+             <img class="s-m-icon" src="images/Vector (3).svg" alt="social-icon">
+         </a>
+        </li>  
+     </ul>
+
+    <button type="button" class="btn">Get my resume</button>
+    </div>
+
+    <div class="skill-sets">
+        <ul>
+            <li>
+                <a href="#">Languages</a>
+                
+                <ul class="language-set">
+                    <li>
+                        <a href="#">
+                            <img src="images/Ellipse 1.png" alt="Html">
+                             <span>Javascript</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="images/css.png" alt="css">
+                            <span>Css</span>
+                        </a>
+                    </li>
+
+                    <li>
+                        <a href="#">
+                            <img src="images/html.png" alt="Html">
+                            <span>Html</span>
+                        </a>
+                    </li>
+                </ul>
+
+                <li class="line"> 
+                    <a href="#">FrameWorks</a>
+                    <span><hr></span>
+                </li>
+                
+                <li class="line">
+                    <a href="#">Skills</a>
+                    <span><hr></span>
+                </li>
+                
+            </li>
+        </ul>
+    </div>
+</section>
```

Comment:
I think your `framework`, `skills`, `html`, `css`, `javascript` and `languages` are not meant to be links since they are not linking you anywhere😉. kindly fix this🤗